spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -311,6 +361,158 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
+        tissue_bridges_df.reset_index(inplace=True)
+
+        # Get slices of minimum dorsal and ventral tissue bridges for each sagittal slice
+        # NOTE: we get minimum because tissue bridges are quantified as the width of spared tissue at the minimum
+        # distance from cerebrospinal fluid to the lesion boundary
+        for sagittal_slice in sagittal_lesion_slices:
+            # Get df for the selected sagittal slice
+            df_temp = tissue_bridges_df[tissue_bridges_df['sagittal_slice'] == sagittal_slice]
+
+            # Get the width of the tissue bridges in mm (by multiplying by p_lst[1]) and use np.cos(self.angles[SLICE])
+            # to correct for the angle of the spinal cord with respect to the axial slice
+            # NOTE: the orientation is RPI (because we reoriented the image to RPI using orient2rpi()); therefore
+            # p_lst[0] is the pixel size in the R-L direction, p_lst[1] is the pixel size in the A-P direction, and
+            # p_lst[2] is the pixel size in the S-I direction.
+            # Since we are computing dorsal and ventral tissue bridges, we use p_lst[1] (A-P direction)
+            dorsal_bridge_width_mm = df_temp.apply(lambda row:
+                                                   row['dorsal_bridge_width'] * p_lst[1] *
+                                                   np.cos(self.angles[row['axial_slice']]), axis=1)
+            ventral_bridge_width_mm = df_temp.apply(lambda row:
+                                                    row['ventral_bridge_width'] * p_lst[1] *
+                                                    np.cos(self.angles[row['axial_slice']]), axis=1)
+
+            # Add the columns to the DataFrame
+            # For some reason I need to add the columns one by one. When I tried to write directly to the DataFrame,
+            # I got the following error:
+            #   "IndexError: only integers, slices (:), ellipsis (...), numpy.newaxis (None) and integer or boolean
+            #   arrays are valid indices"
+            df_temp['dorsal_bridge_width_mm'] = dorsal_bridge_width_mm
+            df_temp['ventral_bridge_width_mm'] = ventral_bridge_width_mm
+
+            # Get the axial slices corresponding to the minimum bridge widths
+            # This information is printed to terminal
+            min_dorsal_bridge_width_slice = df_temp.loc[df_temp['dorsal_bridge_width_mm'].idxmin(), 'axial_slice']
+            min_ventral_bridge_width_slice = df_temp.loc[df_temp['ventral_bridge_width_mm'].idxmin(), 'axial_slice']
+
+            # Get the minimum dorsal and ventral bridge widths
+            min_dorsal_bridge_width_mm = float(df_temp[df_temp['axial_slice'] == min_dorsal_bridge_width_slice]
+                                               ['dorsal_bridge_width_mm'])
+            min_ventral_bridge_width_mm = float(df_temp[df_temp['axial_slice'] == min_ventral_bridge_width_slice]
+                                                ['ventral_bridge_width_mm'])
```

Comment:
You can get the minimum value more directly:
```suggestion
            min_dorsal_bridge_width_mm = float(df_temp['dorsal_bridge_width_mm'].min())
            min_ventral_bridge_width_mm = float(df_temp['ventral_bridge_width_mm'].min())
```

Reply:
Good point! Agree. We can commit this suggestion